var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -367,6 +367,12 @@ def determine_b2_options(self, spec, options):
                 cxxflags.append('-stdlib=libc++')
                 options.extend(['toolset=clang',
                                 'linkflags="-stdlib=libc++"'])
+        elif spec.satisfies('%xl_r'):
```

Comment:
I'm guessing this probably needs to be `self.spec.satisfies('%xl') or self.spec.satisfies('%xl_r')`. See the `netlib-lapack` package, for example.

Reply:
Hmm, I did this based on usage of just plain `spec.satisfies()` in the same block a few lines up. Any reason it would be needed here but not in the initial `if`?

Reply:
I don't think `self.spec.satisfies` vs. `spec.satisfies` is the issue here. What @AndrewGaspar means is:
```suggestion
        elif spec.satisfies('%xl') or spec.satisfies('%xl_r'):
```

Reply:
Yes, that’s what I meant.

Reply:
alright, I committed the suggestion...